### DISCUSSION TITLE
Unable to visit index page redirect to /sites

### DISCUSSION THREAD

**Discussion by Deep-Codes** [plausible/analytics#1503] [Anything Else]
I want to visit https://plausible.io/

But since I have set up analytics it redirects to https://plausible.io/sites

Is there a way I can visit the index page instead of getting redirected?

---

Also would be great to have Pricing link here in footer 

<img width="1134" alt="Screenshot 2021-12-02 at 12 28 15 AM" src="https://user-images.githubusercontent.com/61158210/144296581-a925929d-e139-4fb6-a2bc-c6c29bb5e041.png">

**Comment by metmarkosaric** [comment]
Thanks @Deep-Codes!

You can log out to visit the home page or use a different Firefox container. This is a change we made last year as it makes it faster for Plausible users to get to their dashboards. There's no plan to revert it as it would introduce one extra click for everyone while there's no benefit for existing users on the home page. 

You can see the pricing at https://plausible.io/#pricing and you can get the pricing in your account settings too. Click on settings and then "upgrade" or "change plan" all depending on your current account status.

**Reply by Deep-Codes** [reply]
That makes sense.

Thanks for answering,
Cheers 😇
